code_of_conduct.md

```diff
@@ -0,0 +1,37 @@
+# Code of Conduct
+
+The Women Who Code DC Mission has three main goals:
+1. That women software engineers feel confident and empowered to pursue their goals.
+1. That every woman codes
+1. That all women who code around Washington DC have a community they can access for support and resources.
+
+It is our responsibility as an organization to cultivate a safer environment for all women (and nonbinary people, too). We are committed to being welcoming and embracing of people of all ages, races, ethnicities, nationalities, sexualities, gender expressions, disabilities, religions, skill levels.
```

Comment:
do we want to address here why we save this space for women?  because we are not really embracing men.. but maybe a proactive statement on why it is so important to have a women's only environment (perhaps also serving as a place people can refer to when they inevitably get asked "why not men")